CDCgov/prime-reportstream
# Cle/7043 juris filter

This PR makes universal pipeline routing use fhirPath instead of being hardcoded.

Test Steps:
1. Run tests
OR
1. Set up a local receiver with filters with topic 'full-elr' and add fhirPath to the jurisdisctionalFilter section
2. Submit a message with client headers for a full-elr sender
3. Observe result

## Changes
- Update organization.yml file with fhirPath filter for co-phd.full-elr-hl7
- Update receiver to allow for jurisdictional filters in test cases
- Update fhirRouter to read and apply the fhirPath. Finds receiver filters first, then org filters if there are no receiver filters
- Added tests

## Checklist

### Testing
